contrib/docker/Dockerfile.compile

```diff
@@ -18,12 +18,17 @@ RUN apt-get -y update \
 
 # EBPF requires llvm-6.0 clang-6.0 however the cross compulation docker image can't install them
 
-RUN PROTOC_VER=3.5.1; \
-    wget --no-verbose https://github.com/google/protobuf/releases/download/v${PROTOC_VER}/protoc-${PROTOC_VER}-linux-$(uname -m).zip \
-    && unzip protoc-${PROTOC_VER}-linux-$(uname -m).zip \
+# need to convert arch ppc64le -> ppcle_64 and aarch64 -> aarch_64
+
+RUN PROTOC_VER=3.7.1; \
+    ARCH=$(uname -m); \
+    if [ $ARCH != x86_64 ]; then ARCH=${ARCH%%64*}${ARCH##*64}_64 ; fi ; \
```

Comment:
ARCH=${ARCH//64/}_64

Reply:
This is a bash only construct and in ubuntu docker build the shell is sh.